eks.tf

```diff
@@ -0,0 +1,45 @@
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "17.24.0"
+
+  cluster_name    = "my-eks"
+  cluster_version = "1.21"
+
+
+  vpc_id     = resource.aws_cloudformation_stack.network.outputs["MyVpc"]
+  subnet_ids = [resource.aws_cloudformation_stack.network.outputs["Subnet1"],
+               resource.aws_cloudformation_stack.network.outputs["Subnet2"]]
```

Comment:
What if i change the name of my subnet
Is the syntax correct?
Please refer to the documentation i shared earlier